Student/src/student/FriendInfo.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package student;
+
+/**
+ *
+ * @author Niaz-Sima
+ */
+public class FriendInfo {

```

Comment:
This class is the same as the other one. Please remote it and use class `Student`.